feat: describe the human subset selection procedure (#50)
lamalab-org/chembench-paper: src/tex/sections/human_subset_selection.tex

> \paragraph{Question selection}

Reviewer: ```suggestion
\paragraph{Question selection} \label{sec:subset-selection}
```

\paragraph{Question selection}

> For selecting a human subset, several design choices have been made. Firstly, from the full dataset, we keep all the questions that are labeled as advanced. In this way, we can obtain a deeper insight about the capabilities of \glspl{llm} on advanced tasks

Reviewer: ```suggestion
Several design choices were made when selecting a human subset. Firstly, from the full dataset, we keep all the questions that are labeled as advanced. In this way, we can obtain a deeper insight about the capabilities of \glspl{llm} on advanced tasks
```

up to you which one you like better

> when compared to actual chemists. Secondly, we sample a maximum of 3 questions across all possible combinations of categories (i.e., knowledge or reasoning) and topics (e.g., organic chemistry, physical chemistry). Thirdly, we do not include any intuition questions

Reviewer: can we still add why we do not include intuition questions?

> when compared to actual chemists. Secondly, we sample a maximum of 3 questions across all possible combinations of categories (i.e., knowledge or reasoning) and topics (e.g., organic chemistry, physical chemistry). Thirdly, we do not include any intuition questions

Reviewer: @AdrianM0 I think this question about a statement about why we exclude intuition is also for you. I think the argument for the human study is simple. The argument why it shouldn't be in the "tiny subset" is not so easy

> when compared to actual chemists. Secondly, we sample a maximum of 3 questions across all possible combinations of categories (i.e., knowledge or reasoning) and topics (e.g., organic chemistry, physical chemistry). Thirdly, we do not include any intuition questions

Reviewer: ```suggestion
when compared to actual chemists. Secondly, we sample a maximum of three questions across all possible combinations of categories (i.e., knowledge or reasoning) and topics (e.g., organic chemistry, physical chemistry). Thirdly, we do not include any intuition questions
```

> in this subset. In total, 248 questions have been sampled for the human study. Then, this subset is divided into 2 subsets based on the aforementioned combinations. One of the question subsets allows tool use, and the other does not.

Reviewer: ```suggestion
in this subset. In total, \variable{output/num_human_answered_questions.txt} questions have been sampled for the human study. Then, this subset is divided into two subsets based on the aforementioned combinations. One of the question subsets allows tool use, and the other does not.
```

Several design choices were made when selecting a human subset. Firstly, from the full dataset, we keep all the questions that are labeled as advanced. In this way, we can obtain a deeper insight about the capabilities of \glspl{llm} on advanced tasks
when compared to actual chemists. Secondly, we sample a maximum of three questions across all possible combinations of categories (i.e., knowledge or reasoning) and topics (e.g., organic chemistry, physical chemistry). Thirdly, we do not include any intuition questions

> in this subset, because the correct answers are subjective, and would hinder an objective evaluation of the chemists against \glspl{llm}. In total, \variable{output/num_human_answered_questions.txt} questions have been sampled for the human study. Then, this set is divided into two subsets based on the aforementioned combinations. One of the question subsets allows tool use, and the other does not.

Reviewer: thanks for including this argument. However, does this argument not raise the question of why we include preference data in the first place?

Author: i do not think so. In a way this argument is to say that the answers for these questions are defined by preference. 
while we want the LLM to align with the human preference, we do not want any preference questions when comparing the LLMs to humans due to the subjectivity.

 and also, the preference questions cannot have a human baseline per say, since they are defined by humans.

Reviewer: I believe the interpretation of the "tiny subset" depends on its intended use. If it's solely for the human study, then the argument is valid—this was, in fact, our original motivation.

However, we also propose the "tiny subset" as a resource for quick and inexpensive evaluation of LLMs. From this perspective, we need to explain why the preference data does not provide a useful signal in this context.

Author: the preference data cannot have a human baseline right? since I feel it is a human baseline itself :D

Reviewer: Yes - this was the argument why we excluded it.

However, we also propose this subset to be used for fast evaluation of LLMs - irrespective of any human baseline. For this usecase, arguments based on a human baseline are tricky or at least indirect.

Author: should I then change the argument to:

"The intutition questions represent a human baseline from a different study. We intend that our "tiny" subset is ued for a fast and fair evaluation of LLMs, independent of any human baselines."

Author: updated the argument
